commit d6bac7d360ac2c8bb3e6a72ca3741c07785279f9
Author: Vasiliy Kulikov <segoon@openwall.com>
Date:   Fri Feb 4 15:24:03 2011 +0300

    platform: x86: tc1100-wmi: world-writable sysfs wireless and jogdial files
    
    commit 8a6a142c1286797978e4db266d22875a5f424897 upstream.
    
    Don't allow everybody to change WMI settings.
    
    Signed-off-by: Vasiliy Kulikov <segoon@openwall.com>
    Signed-off-by: Matthew Garrett <mjg@redhat.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

diff --git a/drivers/misc/tc1100-wmi.c b/drivers/misc/tc1100-wmi.c
index f25e4c9..84733f3 100644
--- a/drivers/misc/tc1100-wmi.c
+++ b/drivers/misc/tc1100-wmi.c
@@ -179,7 +179,7 @@ set_bool_##value(struct device *dev, struct device_attribute *attr, \
 			return -EINVAL; \
 	return count; \
 } \
-static DEVICE_ATTR(value, S_IWUGO | S_IRUGO | S_IWUSR, \
+static DEVICE_ATTR(value, S_IRUGO | S_IWUSR, \
 	show_bool_##value, set_bool_##value);
 
 show_set_bool(wireless, TC1100_INSTANCE_WIRELESS);
